### ISSUE TITLE
tus.Upload with Blob object causes page reload

### ISSUE THREAD

**Issue by MBM1607** [tus/tus-js-client#447]
Calling start on `tus.Upload` object built with `Blob` instead of file causes the page to reload.

I want to upload the blob object taken from the browser user media and upload it directly.

**Setup details**
- Runtime environment: Browser
- Used tus-js-client version: 3.0.0
- Used tus server software: tusd v1.9.1

**Comment by Acconut** [comment]
> submit it when user submits the form but page reload doesn't allow that to occur.

Sorry, I don't understand that. Can you rephrase?

Also, please share your code for using tus-js-client, so we can have a look at it.
